Description: Correct pod NAME issue in LedgerSMB/DBObject/Draft.pm
 Has a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472562&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Draft.pm
+++ b/LedgerSMB/DBObject/Draft.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB:DBObject::Draft, LedgerSMB base class for managing "drafts."
+LedgerSMB:DBObject::Draft - LedgerSMB base class for managing "drafts."
 
 =head1 SYNOPSIS
 
